services/db/migrations/019_cvm.up.sql

```diff
@@ -0,0 +1,29 @@
+create table `cvm_block`
+(
+    block decimal(65) not null primary key,
+    canonical_serialization varbinary(64000) not null,
+    created_at              timestamp        not null default current_timestamp
+);
+
+create table `cvm_transactions`
+(
+    id             varchar(50)     not null primary key,
```

Comment:
Ethereum IDs are actually always exactly 64 characters and addresses are 40. We probably don't need to have a super tight limit but I thought I'd mention it.

Reply:
This id is the hash of the evm.UnsignedExportTx, and evm.UnsignedImportTx bytes..

And that should be a match for the corresponding import/export tx on 'P'/'X' chain..